dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/AnalyticsTableUpdateParams.java

```diff
@@ -166,6 +166,15 @@ public Date getFromDate() {
     return earliest;
   }
 
+  /**
+   * Indicates whether a from date exists.
+   *
+   * @return true if a from date exists.
+   */
+  public boolean hasFromDate() {
```

Comment:
It would sound better if we called it `hasInitialDate`, `hasStartDate`, or something like that.